crypto/tests/test_check.c

```diff
@@ -8347,6 +8348,191 @@ START_TEST(test_compress_coords) {
 }
 END_TEST
 
+START_TEST(test_schnorr_sign_verify) {
+  static struct {
+    const char *message;
+    const char *priv_key;
+    const char *k_hex;
+    const char *s_hex;
+    const char *r_hex;
+  } test_cases[] = {
```

Comment:
Where do these test vectors come from?

If these are your own, can you please also add some that are not your own? Simple google search for "schnorr test vectors" returns some results such as:

* https://raw.githubusercontent.com/sipa/bips/bip-schnorr/bip-schnorr/test-vectors.csv
* https://gist.github.com/kallewoof/5d623445802a84f17cc7ff5572109074

Reply:
I got test vectors from results [this](https://github.com/Zilliqa/Zilliqa/blob/master/src/libCrypto/Schnorr.cpp#L88)  function.
By your first reference they don't use a random K.
By second reference - message in hex format - uncomfortably for me. Need rewrite tests for use it